README.md

```diff
@@ -66,7 +54,7 @@ Follow these instructions to prepare your system for development.
 - Read [TESTING.md](TESTING.md) for more information.
 
 
-## Architecture Summary
+## Legacy Architecture Summary
```

Comment:
Wouldn't it be better to put this somewhere below, since it is legacy? Or add before this some architecture summary which is not legacy?

Also `Phases` section will become legacy too, since there will be no `Update` phase